deploy/olm-catalog/service-binding-operator/0.0.23/service-binding-operator.v0.0.23.clusterserviceversion.yaml

```diff
@@ -5,19 +5,19 @@ metadata:
     alm-examples: |-
       [
         {
-          "apiVersion": "apps.openshift.io/v1alpha1",
-          "kind": "ServiceBindingRequest",
+          "apiVersion": "operators.coreos.com/v1alpha1",
+          "kind": "ServiceBinding",
           "metadata": {
-            "name": "example-servicebindingrequest"
+            "name": "example-servicebinding"
           },
           "spec": {
-            "applicationSelector": {
+            "application": {
               "group": "apps",
               "resource": "deployments",
               "resourceRef": "nodejs-rest-http-crud",
               "version": "v1"
             },
-            "backingServiceSelector": {
+            "services": {
```

Comment:
This should be updated to reflect that `services` is an array

Reply:
Fixed https://github.com/redhat-developer/service-binding-operator/pull/495/commits/ae587fc0b9dc139a570cdcfea5e277a7f1e2ba4f